packages/terra-i18n-plugin/src/I18nAggregatorPlugin.js

```diff
@@ -1,83 +1,86 @@
-import fs from 'fs';
 import path from 'path';
-import mkdirp from 'mkdirp';
 
 let supportedLocales;
 
 function generateTranslationFile(language, messages) {
   return `import { addLocaleData } from 'react-intl';
-import localeData from 'react-intl/locale-data/${language.split('-')[0]}';
+  import localeData from 'react-intl/locale-data/${language.split('-')[0]}';
```

Comment:
I don't think you want the added spacing here. I think it would result in a generated translation file that wouldn't pass the linter.

Reply:
+1, I'd leave spacing as is.